File: src/main/resources/static/js/conexiones/conexion.js
export default class conexion {
    constructor(name,dataLink,vehicle = null){
        this.name = name;
        this.dataLink = dataLink;
        this.vehicle = vehicle;
    }

    setName(name){
        this.name = name;
    }

    setDataLink(dataLink){
        this.dataLink = dataLink;
    }

    setVehicle(vehicle){
        this.vehicle = vehicle;
    }
}

File: src/main/java/mx/hypernova_aero_space/EstacionTerrena/conexiones/conexion.java
package mx.hypernova_aero_space.EstacionTerrena.conexiones;

import mx.hypernova_aero_space.EstacionTerrena.conexiones.AVConnection.AVConnection;
import mx.hypernova_aero_space.EstacionTerrena.conexiones.EDConnection.EDConnection;



public class conexion {

    private String name;
    private EDConnection dataLink;
    private AVConnection vehicle;

    public void synchronizeDataLink(){
        dataLink.synchronize();
    }

    //GETTERS
    public String getName() {
        return name;
    }

    public EDConnection getDataLink() {
        return dataLink;
    }

    public AVConnection getVehicle() {
        return vehicle;
    }

}

// class EnlaceED implements Enlace{

//     //void enviar(){}

// }
